inst/specs/babynames.yml

```diff
@@ -0,0 +1,26 @@
+df:
+  source: babynames::babynames
+  name: nomes
+variables:
+  year:
+    trans: ano
+    desc: "Ano de registro"
+  sex:
+    trans: sexo
+    desc: "Sexo bebê"
+  name:
+    trans: nome
+    desc: nome
+  n:
+    trans: n
+    desc: Número de ocorrências
+  prop:
+    trans: prop
+    desc: "Proporção"
+help:
+  name: bebes
+  alias: bebes
+  title: "Nomes de bebê"
+  description: "Dados completos sobre nomes de bebês fornecidos pela Administração de Segurança Social dos EUA  EE.UU. Inclui todos os nomes com pelo menos 5 usos."
+  usage: nomes
```

Comment:
acho que aqui seria 'bebes', não?

Reply:
seria melhor seguir com acento e definir singular ? "bebê" ?

Reply:
até onde eu entendi, o 'usage:' é como usar a base. por isso usar o padrão. (@rivaquiroga I have a question: i understood that in 'usage' we use the same name of the dataset. is that right?"